server/node/routes.js

```diff
@@ -98,21 +107,25 @@ router.get('/fpx/pi/:id', async (req, res) => {
     const pi = await stripe.paymentIntents.retrieve(req.params.id);
     const charge = pi.charges.data[0];
     const lastPaymentErr = pi.last_payment_error;
+
+    const pm = await stripe.paymentMethods.retrieve(pi.payment_method)
```

Comment:
This is just a question. 
Probably we could get the `account_holder_type` from `charge.payment_method_details.fpx`
https://stripe.com/docs/api/charges/object?lang=node#charge_object-payment_method_details . 

Avoid extra retrieval.

Reply:
Yup, turns out you can. Thanks for the catch!